deployment_config/dev_vars.yml

```diff
@@ -1 +1,6 @@
+AUTH_CLIENT_ID: ${AUTH_CLIENT_ID}
```

Comment:
It looks like these are getting sent as literal values, not expanded to what is set in CircleCI: https://dashboard.fr.cloud.gov/applications/2oBn9LBurIXUNpfmtZCQTCHnxUM/ce39fffc-0a4f-4b06-a202-4ab147effb64/variableshttps://dashboard.fr.cloud.gov/applications/2oBn9LBurIXUNpfmtZCQTCHnxUM/ce39fffc-0a4f-4b06-a202-4ab147effb64/variables

Reply:
Excellent catch!